module/move/willbe/src/package.rs

```diff
@@ -395,7 +395,7 @@ mod private
   ///
   /// Returns:
   /// Returns a result containing a report indicating the result of the operation.
-  pub fn publish_single( package : &Package, force : bool, dry : bool ) -> Result< PublishReport, ( PublishReport, wError ) >
+  pub fn publish_single( package : &Package, force : bool, dry : bool, base_temp_dir : Option< &Path > ) -> Result< PublishReport, ( PublishReport, wError ) >

```

Comment:
put all arguments into map of options ( special struct with all arguments named ) 
use maybe former?